.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-    - "0.10"
+    - "0.12"
```

Comment:
Might as well go big and do v4 or v6. The pre-v1 versions aren't supported by the node team anymore.